charts/prl-citizen-frontend/values.preview.template.yaml

```diff
@@ -22,6 +22,8 @@ nodejs:
     CDAM_API_URL: 'https://ccd-case-document-am-api-prl-ccd-definitions-pr-676.service.core-compute-preview.internal'
     COS_URL: 'https://prl-cos-pr-562.service.core-compute-preview.internal'
     REDIS_HOST: 'prl-citizen-frontend-redis-aat.redis.cache.windows.net'
+    LAUNCH_DARKLY_OFFLINE_MODE: false
+    C100_REBUILD_FLAG: false
```

Comment:
@yogendraU Can we enable it in preview env by default ?

Reply:
C100_REBUILD_FLAG has no significance in preview as flag is being calculated based on if LAUNCH_DARKLY_OFFLINE_MODE = false, then it fetches the flag from Launch Darkly API